TROUBLESHOOTING.md

```diff
@@ -1,82 +1,64 @@
-Troubleshooting
-=====
 
-Common Issues
-----
+# Troubleshooting
+
+
+## Common Issues
+
 ### AngularJS
-If you're testing against an AngularJS app, you may have trouble testing against the correct page when you make use of ngView to render templates. We recommend using [`actions`][actions] to wait until a particular URL fragment has loaded, or for an element to become available on the page.
 
-You could also write a custom `beforeScript` [PhantomJS hook][phantom-hook] to capture an [emitted Angular view load event][ngViewLoaded]. 
+If you're testing against an AngularJS app, you may have trouble testing against the correct page when you make use of ngView to render templates. We recommend using [`actions`][actions] to wait until a particular URL fragment has loaded, or for an element to become available on the page.
 
-You could also use a timeout to give your view time to load, but this is unreliable. 
+You could also use a timeout to give your view time to load, but this is unreliable.
 
 ### Why does Pa11y give different results to the HTML CodeSniffer bookmarklet?
-Pa11y uses PhantomJS as a headless web browser to load the DOM content and can only analyze what is provided.
+
+Pa11y uses Headless Chrome as a web browser to load the DOM content and can only analyze what is provided.
+
 If parts of the DOM are been loaded after the document is first generated, you may get results that differ from the bookmarklet which runs in the browser and can test against the complete DOM.
 
 If you use Pa11y and HTML CodeSniffer CLI you will find that you get the same results, which will both differ from the bookmarklet, a similar issue was highlighted by [HTML CodeSniffer][sniff-issue].
 
 ### Why does Pa11y give different results each time it runs?
+
 This could be a number of things, but indicates that the content of your page changes in some way on each load. If you include advertising on your page then you can expect to see differing results, and sometimes your JavaScript may not have had time to execute before the Pa11y test runs.
 
 To debug this, you can use the `--screen-capture` flag or [`screenCapture` option](https://github.com/pa11y/pa11y#screencapture-string) and compare the output for the differing test runs.
 
 ### Pa11y ignores the contents of iframes
-Pa11y doesn't currently support testing against iframe content from a parent page context. Any page that makes use of iframes, e.g. for displaying ads, may show different error, warning, and notice counts in Pa11y compared to the HTML_CodeSniffer browser bookmarklet.
-
-If you do need to test the contents of an iframe, run Pa11y against the iframe source URL directly.
 
-### PhantomJS fails to run and returns exit code 127
+TODO verify that this is still an issue with Headless Chrome
```

Comment:
We won't know if headless chrome has similar problems until it's being used out in the wild - even if it does have similar problems, they won't really look like this one. 

Maybe we should leave this section alone so that 4.x users can still refer to it, until 4.x is retired